darkmoon/api/v1/metadata/views.py

```diff
@@ -219,7 +216,6 @@ async def upload_metadata(
             uses invalid characters
         ServerNotFoundException:
             Endpoint is unable to connect to mongoDB instance
```

Comment:
This should be under a raises section along with the other exceptions for this endpoint